internal/common/disasters/setup.go

```diff
@@ -0,0 +1,22 @@
+package disasters
+
+import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/config"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+var envConf = config.GameConfig().DisasterConfig
```

Comment:
hmm no sure what I think of this -- makes it mutable, I'd prefer it to be actually called when used. 

As an aside, passing required configs as a parameter to functions makes your functions easily testable without reliance on the config.

Reply:
Yeah that's a good point - guess it kind of defeats the point of wrapping it in a function in the first place.

And yeah also true about testing. I initially did it this way because it seemed neater and thought the function signature would just become unwieldy with config as params. But yeah hadn't done tests for this yet so hadn't thought about that part! Will change